ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/TransactionBubble.qml

```diff
@@ -0,0 +1,128 @@
+import QtQuick 2.3
+import "../../../../../shared"
+import "../../../../../imports"
+import "./TransactionComponents"
+
+Rectangle {
+    property string tokenAmount: "100"
+    property string token: "SNT"
+    property string fiatValue: "10 USD"
+    property bool outgoing: true
+    property string state: "addressReceived"
+    property string time: "9:41 AM"
```

Comment:
What do you think of changing `time` to be a numeric value, and then format it in the `value` of the styledtext `timeText`?

Reply:
Good idea